Fix EventLog remark (#18998)
dotnet/AspNetCore.Docs: aspnetcore/fundamentals/logging/index.md

### Windows EventLog

The `EventLog` provider sends log output to the Windows Event Log. Unlike the other providers, the `EventLog` provider does ***not*** inherit the default non-provider settings. If `EventLog` log settings aren't specified, they [default to LogLevel.Warning](https://github.com/dotnet/extensions/blob/release/3.1/src/Hosting/Hosting/src/Host.cs#L99-L103).

> To log events lower than <xref:Microsoft.Extensions.Logging.LogLevel.Warning?displayProperty=nameWithType>, explicitly set the log level. In the following example, the Event Log default log level is set to <xref:Microsoft.Extensions.Logging.LogLevel.Information?displayProperty=nameWithType>:

Reviewer: ```suggestion
To log events lower than <xref:Microsoft.Extensions.Logging.LogLevel.Warning?displayProperty=nameWithType>, explicitly set the log level. The following example sets the Event Log default log level to <xref:Microsoft.Extensions.Logging.LogLevel.Information?displayProperty=nameWithType>:
```

* `SourceName`: ".NET Runtime"
* `MachineName`: The local machine name is used.

> Events are logged for [Warning level and higher](#log-level). To log events lower than <xref:Microsoft.Extensions.Logging.LogLevel.Warning?displayProperty=nameWithType>, explicitly set the log level. In the following example, the Event Log default log level is set to <xref:Microsoft.Extensions.Logging.LogLevel.Information?displayProperty=nameWithType>:

Reviewer: ```suggestion
Events are logged for [Warning level and higher](#log-level). The following example sets the Event Log default log level to <xref:Microsoft.Extensions.Logging.LogLevel.Information?displayProperty=nameWithType>:
```
